### ISSUE TITLE
Closed expansion panel will focus hidden elements

### ISSUE THREAD

**Issue by csvn** [angular/material2#5432]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
If an `md-expansion-panel` has focus and is closed, the next item to gain focus when pressing tab should be the next `md-expanion-panel`.

#### What is the current behavior?
If a closed and focused `md-expansion-panel` has any buttons (or similar focusable targets), tab will go to those elements. This occurs even though the elements are not visible on the page.

#### What are the steps to reproduce?
https://github.com/angular/material2/blob/master/src/demo-app/expansion/expansion-demo.html

The expansion panel demo should be enough to reproduce:

1. Focus the closed **"Single expansion panel"**
2. Press tab
3. Focus will land on the `cancel` button

#### What is the use-case or motivation for changing an existing behavior?
Usability and expected behavior.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular Material `v2.0.0-beta.7`.

#### Is there anything else we should know?
I don't think so, but tell me if I missed something.
